test/parallel/test-dgram-bind-shared-ports.js

```diff
@@ -64,10 +64,10 @@ if (cluster.isMaster) {
 
   socket1.bind({
     address: 'localhost',
-    port: common.PORT,
+    port: 0,
```

Comment:
If I'm reading the test correctly this can't be a dynamic port as the same port should be used by the two workers.